resources/geometry.rb

```diff
@@ -517,48 +507,43 @@ def self.process_occupants(model, runner, num_occ, occ_gain, sens_frac, lat_frac
-        if not people_sch.validated?
-          return false
-        end
-      end
+    space_num_occ = num_occ * UnitConversions.convert(space.floorArea, "m^2", "ft^2") / cfa
 
-      if activity_sch.nil?
-        # Create schedule
-        activity_sch = OpenStudio::Model::ScheduleRuleset.new(model, activity_per_person)
+    if people_sch.nil?
+      # Create schedule
+      people_sch = MonthWeekdayWeekendSchedule.new(model, runner, Constants.ObjectNameOccupants + " schedule", weekday_sch, weekend_sch, monthly_sch, mult_weekday = 1.0, mult_weekend = 1.0, normalize_values = true, create_sch_object = true, schedule_type_limits_name = Constants.ScheduleTypeLimitsFraction)
+      if not people_sch.validated?
+        return false
       end
+    end
 
-      # Add people definition for the occ
-      occ_def = OpenStudio::Model::PeopleDefinition.new(model)
-      occ = OpenStudio::Model::People.new(occ_def)
-      occ.setName(space_obj_name)
-      occ.setSpace(space)
-      occ_def.setName(space_obj_name)
-      occ_def.setNumberOfPeopleCalculationMethod("People", 1)
-      occ_def.setNumberofPeople(space_num_occ)
-      occ_def.setFractionRadiant(occ_rad)
-      occ_def.setSensibleHeatFraction(occ_sens)
-      occ_def.setMeanRadiantTemperatureCalculationType("ZoneAveraged")
-      occ_def.setCarbonDioxideGenerationRate(0)
-      occ_def.setEnableASHRAE55ComfortWarnings(false)
-      occ.setActivityLevelSchedule(activity_sch)
-      occ.setNumberofPeopleSchedule(people_sch.schedule)
-
-      total_num_occ += space_num_occ
-
-      runner.registerInfo("Space '#{space.name}' been assigned #{space_num_occ.round(2)} occupant(s).")
+    if activity_sch.nil?
+      # Create schedule
+      activity_sch = OpenStudio::Model::ScheduleRuleset.new(model, activity_per_person)
     end
 
+    # Add people definition for the occ
+    occ_def = OpenStudio::Model::PeopleDefinition.new(model)
+    occ = OpenStudio::Model::People.new(occ_def)
+    occ.setName(space_obj_name)
+    occ.setSpace(space)
+    occ_def.setName(space_obj_name)
+    occ_def.setNumberOfPeopleCalculationMethod("People", 1)
+    occ_def.setNumberofPeople(space_num_occ)
+    occ_def.setFractionRadiant(occ_rad)
+    occ_def.setSensibleHeatFraction(occ_sens)
+    occ_def.setMeanRadiantTemperatureCalculationType("ZoneAveraged")
+    occ_def.setCarbonDioxideGenerationRate(0)
+    occ_def.setEnableASHRAE55ComfortWarnings(false)
+    occ.setActivityLevelSchedule(activity_sch)
+    occ.setNumberofPeopleSchedule(people_sch.schedule)
+
+    total_num_occ += space_num_occ
+
+    runner.registerInfo("Space '#{space.name}' been assigned #{space_num_occ.round(2)} occupant(s).")
+
     runner.registerInfo("The building has been assigned #{total_num_occ.round(2)} total occupant(s).")
```

Comment:
Remove this line (or the line above). Just need to provide the info once now. (Before it was once for each space plus the total for the building.)